modules/rosa-accessing-your-cluster-quick.adoc

```diff
@@ -30,22 +30,24 @@ $ rosa create admin --cluster=<cluster_name>
 W: It is recommended to add an identity provider to login to this cluster. See 'rosa create idp --help' for more information.
 I: Admin account has been added to cluster 'cluster_name'. It may take up to a minute for the account to become active.
 I: To login, run the following command:
-oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \
+oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \ #<1>
 --username cluster-admin \
 --password FWGYL-2mkJI-3ZTTZ-rINns
 ----
+<1> For {hcp-title-first} cluster, the port number should be `443`.
 
 . Enter the `oc login` command, username, and password from the output of the previous command:
 
 +
 .Example output
 [source,terminal]
 ----
-$ oc login https://api.cluster_name.t6k4.i1.oragnization.org:6443 \
+$ oc login https://api.cluster_name.t6k4.i1.oragnization.org:6443 \ #<1>
 >  --username cluster-admin \
 >  --password FWGYL-2mkJI-3ZTTZ-rINns
 Login successful.                                                                                                                                                                                                                                                       You have access to 77 projects, the list has been suppressed. You can list all projects with ' projects'
 ----
+<1> For {hcp-title} cluster, the port number should be `443`.
```

Comment:
Should this read as "For {hcp-title} clusters" or "For a {hcp-title} cluster"?

Might also consider rephrasing for clarity. Something like: "For {hcp-title} clusters, use port 443."

Reply:
Please see [my comment](https://github.com/openshift/openshift-docs/pull/72868#discussion_r1524263756).